web_scraper/data/docs/data.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
Let's not commit this data.  
Also, the output should go in `<project root>/data/docs/`

Reply:
okay, the out put inside the web scraping because i'm giving  the parameter in the command line  while i'm running